backend/app/tests/test_guardrails_api.py

```
@@ -80,6 +81,35 @@ def validate(self, data):
     assert body["error"]
 
 
+def test_rephrase_needed_is_true_when_on_fail_is_rephrase(client, mock_crud):
```

Ruff (0.15.12)

[warning] Unused function argument: `mock_crud`

(ARG001)